Makefile.am

```diff
@@ -1219,10 +1190,12 @@ imap_httpd_SOURCES = \
 	imap/http_applepush.c \
 	imap/http_caldav.c \
 	imap/http_caldav.h \
+	imap/http_caldav_js.h \
```

Comment:
Yep, the foo_js.h files depend on `xxd`, so these ones definitely need to be in the distribution.